ui/eventmenu.py

```diff
@@ -140,6 +153,10 @@ def start(self):
         else:
             self.event.is_awacs_enabled = False
 
+        ca_slot_entry_value = self.ca_slot_entry.get()
+        ca_slots = int(ca_slot_entry_value and ca_slot_entry_value or "0")
```

Comment:
Will raise exception in case of invalid number.

Reply:
Could be a good idea to replace the "Entry" tk widgets by "Spinbox" tk widgets in this window, which allow retrieving int values only unlike the entry widget which can return strings. (http://effbot.org/tkinterbook/spinbox.htm)

The Spinbox widget would also allow capping the max value selected, which is not checked currently.

--------------------

We also have the same exception triggered, if for instance someone enter "a" in any entry in this window and then press the associated "+" button.  Not a big issue though.